src/main.rs

```diff
@@ -276,6 +308,16 @@ fn inner_main() -> Result<ExitCode> {
         configuration.show_source = true;
     }
 
+    if let Some(code) = cli.explain {
+        match explain(&code, cli.format) {
+            Ok(()) => return Ok(ExitCode::SUCCESS),
+            Err(error) => {
+                eprintln!("{}", error);
+                return Ok(ExitCode::FAILURE);
+            }
+        }
+    }
```

Comment:
I think this can be simplified to something like:

```
if let Some(code) = cli.explain {
    explain(&code, cli.format)?;
    return Ok(ExitCode::SUCCESS);
}
```

(The failure code should be handled automatically if `explain` returns `Err`.)

Reply:
Great suggestion! Fixed :)